content/chainguard/chainguard-images/how-to-use-chainguard-images.md

```diff
@@ -23,7 +23,114 @@ Each image has its own usage instructions, which is detailed in their READMEs. A
+
+You can learn about the Chainguard Images tags history in our guide about [Using the Tag History API](chainguard/chainguard-images/using-the-tag-history-api/). 
+
+### Pulling by Digest
+
+Pulling a Chainguard Image by its digest can help improve reproducibility, as it will ensure that you are using the same image each time (versus the tag that may receive updates). 
+
+To pull an image by its digest, you can do so by appending the digest which begins with `sha256`. You can find these on our reference tags history pages.
+
+In our Git example, we can review the [Git Image Tags History](/chainguard/chainguard-images/reference/git/tags_history/) page and choose a relevant digest. We will choose the `:latest-dev` image that was last updated on September 2, 2023 at the time this document was being prepared. 
+
+```sh
+docker pull cgr.dev/chainguard/git@sha256:f6658e10edde332c6f1dc804f0f664676dc40db78ba4009071fea6b9d97d592f
+```
+
+When you pull this image, you'll receive output of the digsest which should match the exact digest you have pulled. 
+
+To learn more about image digests, you can review our video [How to Use Container Image Digests to Improve Reproducibility](/chainguard/chainguard-images/videos/container-image-digests/).
+
+### Specifying Architecture
+
+As Chainguard Images are [built for both AMD64 and ARM64 architecture](/chainguard/chainguard-images/overview/#architecture), you can specify the architecture you would like to use by employing the `--platform` flag with the `docker pull` command. In this example, we'll specify using the `linux/arm64` architecture with the Go image.
+
+```sh
+docker pull --platform=linux/arm64 cgr.dev/chainguard/go
+```
+
+With this image, you can verify the architecture by calling the version.
+
+```sh
+docker run --rm -t cgr.dev/chainguard/go:latest version
+```
+
+You'll receive output similar to the following:
+
+```sh
+go version go1.21.0 linux/arm64
+```
+
+Specifying the platform will ensure that you're using the desired image and relevant architecture.
+
+### Running a Chainguard Image
+
+You can run a Chainguard Image with the `docker run` command. Note that because Chainguard Images are minimalist containers, most of them ship without a shell or package manager. In the case you would like a shell, you can often use the development image, which is tagged as `:latest-dev` (for example, [Python](/chainguard/chainguard-images/reference/python/getting-started-python/) has its dev image at `cgr.dev/chainguard/python:latest-dev`). Otherwise, you can work with Chainguard Images in way similar to other images.
```

Comment:
```suggestion
You can run a Chainguard Image with the `docker run` command. Note that because Chainguard Images are minimalist containers, most of them ship without a shell or package manager. In case you would like a shell, you can often use the development image, which is tagged as `:latest-dev` (for example, [Python](/chainguard/chainguard-images/reference/python/getting-started-python/) has its dev image at `cgr.dev/chainguard/python:latest-dev`). Otherwise, you can work with Chainguard Images in way similar to other images.
```